piet-gpu/shader/path_coarse.comp

```diff
@@ -30,6 +30,7 @@ layout(set = 0, binding = 1) readonly buffer ConfigBuf {
 #define Q_ACCURACY (ACCURACY * 0.1)
 #define REM_ACCURACY (ACCURACY - Q_ACCURACY)
 #define MAX_HYPOT2 (432.0 * Q_ACCURACY * Q_ACCURACY)
+#define MAX_QUADS 16
```

Comment:
Seems a bit low?

Reply:
I think it's adequate, but it probably deserves deeper analysis. In the worst case, the error scales as n^3, so this means the error is 1/4096 what it would be for a single quadratic bezier. The worst case is when the third derivative is large (which happens for S shapes like y=x^3, and probably also cusps). I think it will be fine for basically non-adversarial cases, but it might be worth taking a closer look at adversarial ones.